src/components/application_manager/src/smart_object_keys.cc

```diff
@@ -276,6 +276,8 @@ const char* last_ign_off_time = "last_ign_off_time";
 const char* resume_vr_grammars = "resumeVrGrammars";
 
 const char* ign_off_count = "ign_off_count";
+const char* global_ign_on_counter = "global_ign_on_counter";
+const char* global_ign_off_counter = "global_ign_off_counter";
```

Comment:
remove `global_ign_off_counter`?

Reply:
@yang1070 Agree. Thanks. Will remove it.

Reply:
@yang1070 Removed in bf125f95f4e65b240d46a7a01c8e9bd1a134b983